runtime/commonTest/src/kotlinx/serialization/protobuf/ProtobufMissingFieldsTest.kt

```diff
@@ -0,0 +1,138 @@
+package kotlinx.serialization.protobuf
+
+import kotlinx.serialization.Decoder
+import kotlinx.serialization.Encoder
+import kotlinx.serialization.KSerializer
+import kotlinx.serialization.SerialDescriptor
+import kotlinx.serialization.SerialId
```

Comment:
Please use star-import when a number of imported declarations from a package is more than 3.

Reply:
Ok.